spec/index.bs

```diff
@@ -647,7 +647,7 @@ dictionary GPURenderPassDepthStencilAttachmentDescriptor {
 
     required GPULoadOp depthLoadOp;
     required GPUStoreOp depthStoreOp;
-    required float clearDepth;
+    float clearDepth = 1.0;
```

Comment:
Dzmitry was going to have an alternative proposal for this (so clearDepth is not specified when it's not used). Can we defer this one?

Reply:
Sure.